ntc_templates/templates/edgecore_show_interfaces_brief.textfsm

```diff
@@ -0,0 +1,19 @@
+Value INTERFACE (\S+\s+\d+\s*/\s*\d+)
+Value NAME (.*?)
+Value STATUS (\S+)
+Value PVID (\d+)
+Value PRI (\d+)
+Value SPEED_DUPLEX (\S+)
```

Comment:
Break up into `SPEED` and `DUPLEX` individually

Reply:
Splited it to the 3 new values:
 - negotiation
 - speed
 - duplex